pgrx/src/datum/unbox.rs

```diff
@@ -122,7 +122,10 @@ unsafe impl UnboxDatum for str {
 }
 
 unsafe impl UnboxDatum for &str {
-    type As<'src> = &'src str where Self: 'src;
+    type As<'src>
+        = &'src str
+    where
+        Self: 'src;
```

Comment:
...I would prefer not to?

Reply:
That's how the version of `rustup` in CI formats things.

Reply:
Ugh.

Reply:
Can these have `#[rustfmt::skip]` instead?

Reply:
Hmm... I'll take a look right now.